### ISSUE TITLE
Allow italics in text?

### ISSUE THREAD

**Issue by ejade42** [nowthis/sankeymatic#80]
Hi, I am using this excellent tool to make a flowchart showing the inclusion and exclusion of papers for a meta-analysis, and need to write gene names, which should be styled in italics (*Gene name*). However, I couldn't find an option to do so. I tried markdown (\*Gene name\*) and html (\<i>Gene name\</i>), but neither worked. Is there a way to do this, and if not how taxing would it be to add one?

Many thanks :)

**Comment by eformx** [comment]
Working on a new version of the tool on my local.  It's a font style which current version does not support but it can. Does this look like what you are after. The tool is now available at https://youarehereinfo.com/sankey with italics. Will look at italics for the name only if there is enough interest.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/8d17c2f5-9e92-40b3-847b-2b17f726ac95" />

**Comment by amourata91-bit** [comment]
I am using this great tool to make trophic webs - and I stumbled upon the same issue as the OP. Italics are needed for both sides of my diagram. Any chance of this getting implemented?

**Comment by ejade42** [comment]
Yeah it's not quite right to make all the labels italic rather than specified words e.g. with `*markdown*` or `<i>HTML tags</i>`
